src/test/java/revir/user/command/CreateCommandTest.java

```diff
@@ -0,0 +1,65 @@
+package revir.user.command;
+
+import org.junit.jupiter.api.Test;
+import static org.mockito.Mockito.mock;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import revir.system.Storage;
+import revir.tasks.Deadline;
+import revir.tasks.Event;
+import revir.tasks.TaskList;
+import revir.tasks.Todo;
+import revir.user.Ui;
+
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
```

Comment:
I like the way you included empty lines to separate and structure the import statements which made it a lot clearer.